engine/src/main/java/com/datatorrent/stram/client/AppPackage.java

```diff
@@ -404,6 +407,8 @@ public void processAppDirectory(boolean skipJars)
         LOG.warn("Ignoring file {} with unknown extension in app directory", entry.getName());
       }
     }
+
+    processAppProperties();
```

Comment:
Does moving the processAppProperties here mean that the properties specified in the app json file or the app properties files would be overwritten by the properties-* xml files?